src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/PseRespondToTribunalController.java

```diff
@@ -75,6 +75,40 @@ public ResponseEntity<CCDCallbackResponse> aboutToStart(
         return getCallbackRespEntityNoErrors(caseData);
     }
 
+    /**
+     *  Populates the dynamic list for select an order or request to respond to.
```

Comment:
This javadoc is a bit confusing as its identical to the method above it. I see that one is populating select dropdown whereas the other one is select dropdown view but this doesn't really explain what/where the page is. If these are seperate events then it might be better to have a different controller file?